config/secrets.json.example

```diff
@@ -55,6 +55,7 @@
     "api": {
       "log_requests": true,
       "log_responses": true,
+      "log_user_activity": false,
```

Comment:
We should document this (in `readme.md`?) to say that this should probably be turned on in production but won't work for the tests.